.travis.yml

```diff
@@ -0,0 +1,23 @@
+# ref: https://docs.travis-ci.com/user/languages/python
+language: python
+dist: trusty
+sudo: true
+services:
+- docker
+
+matrix:
+  include:
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+  - python: 3.6
+    env: TOXENV=py36
+  - python: 3.6
+    env: TOXENV=coverage,codecov
```

Comment:
Should we add python 3.7? I think 3.7 is the latest major release and the code looks fine with the latest version.

Reply:
To my memory is correct, @sugyan san tried to use python 3.7 but it didn't work. This is because `async` is not supported in python 3.7.

@sugyan san, could you give us your insight?

Reply:
Yes, there was such a problem. But it only happens with older versions of `kubernetes-client`, and seems to work well with the current version.

- https://github.com/kubernetes-client/python/blob/master/CHANGELOG.md#v700a1
- https://github.com/kubernetes-client/gen/pull/67

Anyway, this repository does not use `kubernetes-client`, so it seems that there is no problem even adding 3.7 to support targets.

Reply:
Thank you @sugyan san!
OK, let’s support Python 3.7.